### PR TITLE
initial sensor model

### PR DESCRIPTION
**Pull request by j-asefa**
Initial model for a generic sensor based on the client's specs (check emails). Wanted to include it mostly for documentation.

### CODE REVIEW

File: kaos/sensors/sensor.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                pointing_horizontal_half_angle,
+                pointing_vertical_half_angle,
+                scene_duration,
+                sensor_pointing_type,
+                sensor_type,
+                stabilization_time,
+                view_horizontal_half_angle,
+                view_vertical_half_angle):
+
+        self.sensor_id                          = sensor_id
```
Comment (lsav) [inline kaos/sensors/sensor.py]: Please don't align variable assignments. (makes more work in the future when changing/adding things)

File: kaos/sensors/sensor.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                                    edge of the sensor in the XZ plane of the sensor's coordinate system.
+        view_vertical_half_angle: The angle from the boresight (Z) direction to the
+                                    edge of the sensor in the YZ plane of the sensor's coordinate system.
+
+    Example attribute string (Note that the attributes of this string are out of order with the above):
+
+    '831cf482-b37b-4468-83cd-659814256892',true,,'0','0','100','1',
+    'WV110','41fe7f26-c557-49eb-aec8-dd658966f3a6','40.6','-1','-1','0',0,1,'1','2','0.6'
+    """
+    def __init__(self,
```
Comment (lsav) [inline kaos/sensors/sensor.py]: @KMC-70/kaos-overlords Since the sensors seem to be provided in JSON, I'm wondering if we should just use the json library to parse them automatically. Would save time vs having to instantiate with almost 20 different fields) and is just as easy to access. Comments?

Comment (ZeyadTamimi) [reply kaos/sensors/sensor.py]: Couple of comments on that:
1) Sensor data is actually provided in their PSQL database not a database. 

2) Having a constructor with fields is fine. This will go away once we subclass `flask.db.Model`. It will create the default constructor for us that takes all these fields as optional parameters.

3) The json thing is separate discussion, we have to create a serializer/deserializer to automatically handle this conversion between json text and python object representations of the database records.

TLDR: This is fine for now

Comment (lsav) [reply kaos/sensors/sensor.py]: In that case, I'd suggest we leave the sensors off until the db is set up.

This file would be good as a README, though, to document what sensor models ought to contain.
